rules/brand-more-than-logo/rule.md

```diff
@@ -0,0 +1,74 @@
+---
+type: rule
+title: Do you know your brand is more than your logo?
```

Comment:
Hi @bettybondoc 

I've been trying to avoid "your" in rules...

- [ ] 1. What about changing the title to "Do you know branding is more than a logo?"